test/onigumo_test.exs

```diff
@@ -3,19 +3,18 @@ defmodule OnigumoTest do
   import Mox
 
   @urls [
-    "http://onigumo.local/hello.html",
-    "http://onigumo.local/bye.html"
+    {"http://onigumo.local/hello.html", "aHR0cDovL29uaWd1bW8ubG9jYWwvaGVsbG8uaHRtbA"},
+    {"http://onigumo.local/bye.html", "aHR0cDovL29uaWd1bW8ubG9jYWwvYnllLmh0bWw"}
```

Comment:
This is probably not a good solution. We don’t want to test the encoding algorithm here. We trust the `Base` module and can use it in the test too. It makes the tests more complex as well. Using a Struct would be even better, but that would again increase complexity.